migrations/20201123194206-create-events-users.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Events_Users', {
+      id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        allowNull: false
```

Comment:
this is used as foreign key. You need to specify this in migration. This field should also include `references` prop. Smthng like: 
```
references: {
          model: 'Users',
          key: 'id',
        },
```

Reply:
this comment relates to all fields that actually are foreign key.

Reply:
You can update this by writing new migration:)